src/ape/api/transactions.py

```diff
@@ -30,7 +31,7 @@ class TransactionAPI(BaseInterfaceModel):
     chain_id: int = Field(0, alias="chainId")
     receiver: Optional[AddressType] = Field(None, alias="to")
     sender: Optional[AddressType] = Field(None, alias="from")
-    gas_limit: Optional[GasLimit] = Field(None, alias="gas")
```

Comment:
Should we delete this type from `ape.types`? I think this is was the only reason it was created.

Reply:
```
src/ape_ethereum/ecosystem.py:    gas_limit: GasLimit = "auto"
src/ape_ethereum/ecosystem.py:    def validate_gas_limit(cls, value: GasLimit) -> GasLimit:
src/ape/api/networks.py:    def gas_limit(self) -> GasLimit:
```

Reply:
I wanted to move this Union to the outer limit of validating when `kwargs` first come in so that `TransactionAPI` can be simple, might need to do some work though